### DISCUSSION TITLE
Decompilation of SubFolders

### DISCUSSION THREAD

**Discussion by jogibear9988** [icsharpcode/ILSpy#2663] [General] [answered]
Somtimes IL Spy Decompiles Classes to Folders like this:

"aa.bb.cc.dd" and sometimes "aa\bb\cc\dd" (subfolders) are used.

Has this changed during releases? I don't know why some of my decompilations ar elike the first, some are like the second

**Comment by greenozon** [comment, marked as answer]
It's controlled by options in ILSpy:
![image](https://user-images.githubusercontent.com/765343/160585934-047c45f2-7e93-43b5-87c2-fb134e2e42d5.png)
